src/views/templates/actions/editBootSource.ts

```diff
@@ -79,14 +81,29 @@ export const getBootDataSource = async (
     );
 };
 
+export const getDataSourceDataVolume = async (
+  dataSourcePVCName: string,
+  dataSourcePVCNamespace: string,
+): Promise<V1beta1DataVolume | undefined> => {
+  let dataVolume: V1beta1DataVolume = null;
+
+  try {
+    dataVolume = await getDataVolume(dataSourcePVCName, dataSourcePVCNamespace);
+  } catch (error) {
+    console.warn(error);
```

Comment:
Not sure if we really want `console.warn` here, if it's about catching error. 
Also maybe we want to display the error/warning the different way - visibly for the user - as in other places in the code... Can you, please, explain the reason of using `console.warn` here? Just for better understanding of this change. Thanks! :)

Reply:
I used warn because deep source wants something in the catch. If there is no dataVolume it's not an error. Just means that there is no DV

Reply:
Hmm, maybe `console.warn` + some comment in the code about why we use `warn` and not `error`, could be sufficient :)

Reply:
Maybe if the client lost connection? So also the creation will fail.